modules/kubernetes-addons/agones/locals.tf

```diff
@@ -19,7 +18,8 @@ locals {
 
   helm_config = merge(
     local.default_helm_config,
-    var.helm_config
+    var.helm_config,
+    { values = distinct(concat(try(var.helm_config["values"], []), local.default_helm_values)) }
```

Comment:
can you pls break a line for readability

Reply:
Yes sure, just push the changes